mtp_api/apps/prison/models.py

```diff
@@ -9,3 +10,19 @@ class Prison(TimeStampedModel):
 
     def __str__(self):
         return self.name
+
+
+class PrisonerLocation(TimeStampedModel):
+    created_by = models.ForeignKey(settings.AUTH_USER_MODEL)
+
+    prisoner_number = models.CharField(max_length=250)
+    prisoner_dob = models.DateField()
+
+    prison = models.ForeignKey(Prison)
+
+    def __str__(self):
+        return 'location record ' \
+            'for prisoner {prisoner_number} / {prisoner_dob}'.format(
+                prisoner_number=self.prisoner_number,
+                prisoner_dob=self.prisoner_dob
+            )
```

Comment:
When're we likely to use this? I wonder if it's worth not having (or having a method that explicitly doesn't return prisoner details), in case we end up leaking prisoner locations into our logs?